app/src/main/java/com/kamron/pogoiv/updater/AppUpdateLoader.java

```diff
@@ -27,10 +26,19 @@ public void run() {
         try {
```

Comment:
As API endpoint we could replace `https://api.github.com/repos/farkam135/GoIV/releases` with an endpoint for the latest version, `https://api.github.com/repos/farkam135/GoIV/releases/latest`.
https://developer.github.com/v3/repos/releases/#get-the-latest-release